provisions/roles/openshift/tasks/config.yml

```diff
@@ -1,15 +1,17 @@
 ---
 - name: Enable Docker insecure registry
-  replace: >
-    dest=/etc/sysconfig/docker
+  lineinfile:
+    path=/etc/sysconfig/docker
+    state=present
     regexp="^#*\s*INSECURE_REGISTRY=.*"
-    replace='INSECURE_REGISTRY="--insecure-registry 172.30.0.0/16 --insecure-registry {{ intranet_registry }}"'
+    line='INSECURE_REGISTRY="--insecure-registry 172.30.0.0/16 --insecure-registry {{ intranet_registry }}"'
 
 - name: Enable Docker registry
-  replace: >
-    dest=/etc/sysconfig/docker
+  lineinfile:
+    path=/etc/sysconfig/docker
+    state=present
     regexp="^#*\s*ADD_REGISTRY=.*"
```

Comment:
Same comment as for `INSECURE_REGISTRY`. There's nothing like `# ADD_REGISTRY` in the `/etc/sysconfig/docker` file.

Reply:
sure updating the same